Gemfile

```diff
@@ -6,4 +6,5 @@ gem 'jekyll-archives', group: :jekyll_plugins
 gem 'nokogiri', group: :jekyll_plugins
 gem 'html-proofer', group: :jekyll_plugins
 gem 'jekyll-redirect-from', group: :jekyll_plugins
-gem 'webrick', group: :jekyll_plugins
\ No newline at end of file
+gem 'webrick', group: :jekyll_plugins
+gem 'jekyll-toc', group: :jekyll_plugins
```

Comment:
This is not a supported plugin for GitHub page: https://pages.github.com/versions/